src/app/views/others/failover/failover.component.ts

```diff
@@ -49,6 +54,9 @@ export class FailoverComponent implements OnInit {
           });
       },
       complete: () => { // show reboot screen
+        this.window.sessionStorage.setItem('ha_status', 'false');
```

Comment:
Why are we clearing HA status on failover?

Also, setting `ha_status` would belong to an effect and you'd be dispatching a different action from here, not `haStatusLoaded`.